ui/src/components/VideoOverlay.tsx

```diff
@@ -10,7 +10,7 @@ import LoadingSpinner from "@components/LoadingSpinner";
 import Card, { GridCard } from "@components/Card";
 
 interface OverlayContentProps {
-  children: React.ReactNode;
+  readonly children: React.ReactNode;
```

Comment:
There's a warning in VS Code that all these properties should be declared `readonly` (which they should), so while I was in here I updated them. We need to do a sweep through the UI to get the rest... another PR will arrive for ONLY warning-fixes.